### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by ntazelo**
### Features added

**A mobile version with a Hamburger button to open the menu**
**A desktop version with a navbar with a position fixed**
**A section dedicated to the program**
**A section dedicated to the speaker**
**A section dedicated to the sponsors**
**A footer section for the home and about page**
**A section dedicated to the logo in the about page**
**A section dedicated to the history event in the about page**

**A link to the video presentation [link](https://www.loom.com/share/8f7d2d9f879a4b079d8eebe1b404d333)**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+          <li class="mob-single-link"><a href="#" class="mob-link">Join</a></li>
+          <li class="mob-single-link">
+            <a href="#" class="mob-link">Sponsor</a>
+          </li>
+          <li class="mob-single-link"><a href="#" class="mob-link">News</a></li>
+          <li class="mob-single-link">
+            <a href="#" class="mob-link">CC Campaign</a>
+          </li>
+        </ul>
+      </nav>
```
Comment (GzimAsani) [inline about.html:40]: - [ ] Please follow closely the design guidelines, in the navbar give more margin/padding to the buttons

Yours:
Screenshot from 2021-10-15 19-17-35 (copy)

Original:
Screenshot from 2021-10-15 19-19-40

File: index.html [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+          incidunt delectus optio illo ducimus vero recusandae minus doloremque
+          ex repellat deserunt ipsa cupiditate quos, dolore facere voluptas
+          facilis non reiciendis?
+        </p>
+        <h2 class="intro-date">2021.10.11(THU) ~ 16(FRI)</h2>
+        <h5 class="intro-location">
+          @ Lorem, ipsum dolor sit amet consectetur adipisicing.
+        </h5>
+      </div>
+    </section>
```
Comment (GzimAsani) [inline index.html:121]: - [ ] Please follow closely the design guidelines in the margin, padding, size, and color

Yours:
Screenshot from 2021-10-15 19-17-35
Original:
Screenshot from 2021-10-15 19-17-39

File: index.html [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+      <div class="main-wrapper">
+        <h1 class="main-title">Main Program</h1>
+        <ul class="activity-cont">
+          <li class="single-activity-wrapper">
+            <ul class="activity-prog-cont">
+              <li class="activity-prog">
+                <i class="fas fa-sun fa-3x"></i>
+              </li>
+              <li class="activity-prog">
+                <h2 class="activity-prog-title">Global Warming</h2>
```
Comment (GzimAsani) [inline index.html:134]: - [ ] Based on design guidelines this header should have red color

Yours:


Screenshot from 2021-10-15 19-24-40
Original:
Screenshot from 2021-10-15 19-24-34

File: index.html [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+                  Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                  Repudiandae accusamus cum itaque at tempora?
+                </p>
+              </li>
+            </ul>
+          </li>
+        </ul>
+        <a href="#" class="btn-action-reg" id="join"
+          >Join World Environment Summit</a
+        >
```
Comment (GzimAsani) [inline index.html:212]: - [ ] Please follow design guidelines here

Yours:
Screenshot from 2021-10-15 19-26-34
Original:
Screenshot from 2021-10-15 19-26-35

File: about.html [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+            Lorem ipsum dolor sit amet consectetur adipisicing.
+            <a href="#" class="url-cont"
+              >https://www.behance.net/gallery/29845175/CC-Global-Summit-2021</a
+            >
+            Lorem ipsum dolor sit amet.
+          </p>
+          <p class="copyright-owner">&copy; Copyright Earth Planet 2021</p>
+        </div>
+      </div>
+    </footer>
```
Comment (GzimAsani) [inline about.html:229]: - [ ] Footer should contain a logo aligned to the left and text aligned to the right. Check the picture below

Yours:
Screenshot from 2021-10-15 19-30-45
Original:
Screenshot from 2021-10-15 19-30-48

File: about.html [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+        </p>
+        <p class="about-contact">
+          Please contact us per Email for any further questions about WE Global
+          Summit 2021!
+        </p>
+        <a href="#" class="about-contact-link">
+          @ Lorem, ipsum dolor sit amet consectetur adipisicing.
+        </a>
+      </div>
+    </section>
```
Comment (GzimAsani) [inline about.html:123]: - [ ] Please adjust this section to match the original font, color, size, margin/padding

Yours:
Screenshot from 2021-10-15 19-33-38
Original:
Screenshot from 2021-10-15 19-33-46

File: about.html [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+          <p class="society-social-part">
+            Lorem ipsum dolor sit amet consectetur adipisicing.
+            <a href="#" class="url-cont"
+              >https://www.behance.net/gallery/29845175/CC-Global-Summit-2021</a
+            >
+            Lorem ipsum dolor sit amet.
+          </p>
+          <p class="copyright-owner">&copy; Copyright Earth Planet 2021</p>
+        </div>
+      </div>
```
Comment (GzimAsani) [inline about.html:228]: - [ ] This part in the `about` page should not be visible in the desktop version
Screenshot from 2021-10-15 19-37-03

File: index.html (hunk 3 above)
Comment (GzimAsani) [reply index.html:134]: - [ ] In the mobile version the picture and text alignment should stretch the same across each of them

Yours:
Screenshot from 2021-10-15 19-41-43
Original:
Screenshot from 2021-10-15 19-39-48

File: about.html (hunk 5 above)
Comment (GzimAsani) [reply about.html:229]: - [ ] Also for the mobile version the footer should contain a logo aligned to the left and text aligned to the right. Check the picture below
Yours:
Screenshot from 2021-10-15 19-45-03
Original
Screenshot from 2021-10-15 19-45-27
:

File: index.html [hunk 8]
```diff
@@ -0,0 +1,265 @@
...
+        <div class="navbar-wrapper">
+          <a href="./index.html" class="link-page">
+            <figure class="logo-img-cont">
+              <img
+                src="./images/logo.png"
+                alt="logo"
+                class="logo-footer logo-navbar"
+              />
+              <div class="logo-text">
+                <h1 class="website-name logo-name">Earth</h1>
```
Comment (GzimAsani) [inline index.html:74]: - [ ] There should be no more than 1, `h1` tag in each page. _Currently you have **8** in your index.html page and **8** in your about page_

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @ntazelo     
Good job so far! Your portfolio looks great :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good use of gitflow :+1: 
- I like the hover effect of your hamburger button :+1: 

## Required Changes ♻️ 
- [ ] Add a more descriptive PR and be more descriptive about your work and changes that you've made
- [ ] Add live link in the readme file and use GitHub pages
- [ ] Your hamburger menu should not have a fixed position
- [ ] _Check inline comments for more information :thought_balloon: _


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (sinansevgi) [reply index.html]: - [ ] You should follow the design guidelines given in the template. The yellow color you use is suits better with the theme that you choose for the project. :disappointed: However, We expect you to personalize only the content of the page, not the design itself. So as my previous code reviewer requested, I have to ask you to follow the design principles and change the color of these `yellow` parts to the original colors given in the design guidelines.

File: about.html (hunk 1 above)
Comment (sinansevgi) [reply about.html:40]: - [ ] I can't see any commits or change about this issue. Please adjust the margin/padding as requested.

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
------

Hi @ntazelo , 

I really like the topic that you choose. :+1: 
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Please check the inline comments for my suggestions_ :arrow_down:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Happy Hacking! :computer: 

-----

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @ntazelo , 

Great Work :+1: 
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**